frontend/src/pages/Login.tsx

```diff
@@ -13,9 +16,7 @@ export default function Login() {
         <Typography variant="h1" gutterBottom>
           PeerPrep
         </Typography>
-        <TextField id="outlined-basic" label="Username" variant="outlined" />
-        <TextField id="outlined-basic" label="Password" variant="outlined" />
-        <Button href={`/dashboard`} variant="contained">
+        <Button onClick={auth.redirectToSignIn} variant="contained">
```

Comment:
Can we remove this now that the NavBar contains the login/logout button?

Reply:
Will be addressed in separate PR